tasmota/language/af_AF.h

```diff
@@ -651,7 +651,13 @@
 #define D_SENSOR_SPI_MISO      "SPI MISO"
 #define D_SENSOR_SPI_MOSI      "SPI MOSI"
 #define D_SENSOR_SPI_CLK       "SPI CLK"
-#define D_SENSOR_BACKLIGHT     "Backlight"
+#define D_SENSOR_SDIO_CMD      "SDIO CMD"
+#define D_SENSOR_SDIO_CLK      "SDIO CLK"
+#define D_SENSOR_SDIO_D0       "SDIO D0"
+#define D_SENSOR_SDIO_D1       "SDIO D1"
+#define D_SENSOR_SDIO_D2       "SDIO D2"
+#define D_SENSOR_SDIO_D3       "SDIO D3"

```

Comment:
Can't we use "SDIO D" and then index by number like we do for relays and buttons?

Reply:
I thought about it at first, but it has two major drawbacks:
- it would prevent having a second SDMMC connected. ESP32 has 2 slots for SDMMC
- GPIO numbers starts at `1` in Tasmota but the naming in SDIO is `D0`. It would be very confusing to have `SDIO D-1` mapped to SDIO D0.

Reply:
Convinced